week-1/interpret/percentage-change.js

```diff
@@ -9,12 +9,32 @@ const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+<<<<<<< HEAD
+// There are 3. Line 4, Line 7 and Line 10.
+
+// b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+=======
 // b) Identify all the lines that are variable reassignment statements
+>>>>>>> 03fce56db903fb986ff36f73e4d57cebfa18707c
 
 // c) Identify all the lines that are variable declarations
 
+<<<<<<< HEAD
+// There are 2. On line 4 and line 5
+
+// d) Identify all the lines that are variable declarations
+
+// There are 2. On line 7 and line 8
```

Comment:
Lines 7 and 8 use the `const` declaration so these are not stricty variables.  They are constants.  Lines 1 and 2 declare variables using the `let` command.  The difference is that variables can have their values changed but constants are only ever assigned one value